internal/generateTypeScript.ts

```diff
@@ -26,7 +41,15 @@ export const DURATION_TS = `export type Duration = {
 };
 `;
 
-export function generateTypeScript(schema: FoxgloveSchema): string {
+export type GenerateTypeScriptOptions = {
+  /** Include TypedArray types on numeric array fields, e.g. `number[] | Uint32Array` */
```

Comment:
Is it worth mentioning when it might be important to use this?

Reply:
updated comment